api/service.py

```
@@ -819,6 +866,122 @@ async def get_replay_session(incident_type: str = "VOLTAGE_SPIKE"):
+
+    try:
+        # Train models
+        metrics = await predictive_engine.train_models()
+
+        return {
+            "status": "training_completed",
+            "metrics": metrics,
+            "timestamp": datetime.now()
+        }
+
+    except Exception as e:
```

Ruff (0.14.10)

Do not catch blind exception: `Exception`

(BLE001)